lib/streamlit/elements/media_proto.py

```diff
@@ -24,6 +24,8 @@
 import base64
 import re
 
+from six import string_types
+
```

Comment:
Actually this gets imported automatically by some black magic inside `streamlit.compatibility`.

About `compatibility.py`: That file has some really ugly code we totally need to get rid of. But rather than spend time cleaning up that code, a few months ago we decided to basically just wait until Python 2 goes away for good so we can remove all that craziness. In the meantime, I'd rather not start mixing `six` and `compatibility`, which would make things even crazier :wink: 

If you're doing this for lint purposes, can you add some lint ignores on the right lines instead?